dynawo/3rdParty/CMakeLists.txt

```diff
@@ -128,13 +128,22 @@ else()
   set(boost_version        1.64.0)
   set(boost_md5            319C6FFBBECCC366F14BB68767A6DB79)
 endif()
+
+if ( CXX11_ENABLED )  # --------------------------------------------------------------            # Start New part for IIDM-2
+  if(${boost_version} STRLESS "1.65.0")
+    set( boost_version "1.65.0")
```

Comment:
coding style

Reply:
Coding style will be ruled now from a script "dg2cmake". Tell me if it needs any change. New PR is coming soon :)